cypress/e2e/user-journey.cy.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const autosuggestData = require("../../wiremock/src/mappings/search/data/fbi/autosugggest.json");
+const searchResultData = require("../../wiremock/src/mappings/search/data/fbi/searchWithPagination.json");
+const availabilityLabelsData = require("../../wiremock/src/mappings/work/data/fbs/availability.json");
+const workData = require("../../wiremock/src/mappings/work/data/fbi/getMaterial.json");
+const workHoldingData = require("../../wiremock/src/mappings/work/data/fbs/holdings.json");
+const patronData = require("../../wiremock/src/mappings/work/data/fbs/patron.json");
+
+describe("User journey", () => {
+  it("Shows search suggestions & redirects to search result page", () => {
+    cy.createMapping({
+      request: {
+        method: "POST",
+        url: "/opac/graphql",
+      },
+      response: {
```

Comment:
There is one thing I would like to talk with you about looking at these mappings. I find it unnecessary to create the same mappings as we are using in the Pa11y and Lighthouse tests.